lib/calabash.rb

```diff
@@ -23,7 +22,6 @@ module Calabash
 
 
   include Utility
-  include Calabash::API
   include Calabash::Wait
```

Comment:
@TobiasRoikjer This is the direction I want to go.  The gem internals should not depend on the public API.